cypress.json

```diff
@@ -1,5 +1,5 @@
 {
-  "baseUrl": "http://localhost/",
+  "baseUrl": "http://localhost:9000/",
```

Comment:
Isn't the default `localhost:8000` ?

Reply:
it depends if you are testing with `yarn serve` or `yarn develop`. Anyways, it should be either 9000 or 8000, not like it was before, hehe

Reply:
Oh, nice